projects/simulation/SMPL+D_body_models/README.md

```diff
@@ -0,0 +1,60 @@
+# SMPL+D body models
+
+This folder contains code for downloading a large number of human models in SMPL+D body model format, as well as code for making them able to be transferred to Webots for simulations.
+
+<p float="left">
+  <img src="./examples/model_1.png" width=150 />
+  <img src="./examples/model_4.png" width=150 />
+  <img src="./examples/model_3.png" width=150 />
+  <img src="./examples/model_2.png" width=150 />
+</p>
+
+### Download the raw SMPL+D models only.
+```
+python download_data.py raw
+```
+
+### Download the SMPL+D models and convert them as FBX (Filmbox) files. <br/>In this case the pose blend shapes are not applied.
+
+- Download the data by running:
+```
+python src/download_data.py
+```
+
+- Download SMPL for Unity from the official website [here](https://smpl.is.tue.mpg.de/). Once you agree on SMPL license terms and have access to downloads, you will have the following two files:
+```
+SMPL_f_unityDoubleBlends_lbs_10_scale5_207_v1.0.0.fbx
+SMPL_m_unityDoubleBlends_lbs_10_scale5_207_v1.0.0.fbx
+```
+- Place these two files in the ```model``` directory.

```

Comment:
No model directory appears to be included. So should probably mention at what level it should be placed.

Reply:
The 'model' folder is created once the download_data.py has been run. 
https://github.com/tasostefas/opendr_internal/blob/ae7e4a3527d290a3b7524e2af35a24da55874097/projects/simulation/SMPL%2BD_body_models/src/download_data.py#L61